lib/get-selenium-status-url.js

```diff
@@ -35,6 +35,8 @@ exports.getSeleniumStatusUrl = function (seleniumArgs) {
   const nodeConfigArg = seleniumArgs.indexOf('-nodeConfig');
   const portArg = seleniumArgs.indexOf('-port');
   const hostArg = seleniumArgs.indexOf('-host');
+  const isSelenium4 = seleniumArgs[4];
```

Comment:
Please don't rely on index

Reply:
what argument could be used to determine the version? 
I want to use something like `const isSelenium4 = seleniumArgs.indexOf('-version')`

Reply:
let's pass `opts` to `checkStarted` function, then pass it to `getSeleniumStatusUrl`.

Don't use args to detect version, use opts instead

Reply:
I haven't been able to figure out a way to pass opts to checkstarted.
`async function checkStarted(seleniumArgs, opts) {...`
how do I know what value does opts hold here? I think it has something to do with `checkArgs` but I am not sure how to proceed. Could you please advise?

Reply:
you can use debugger and follow the call chain to figure out which function is calling checkStarted function

![image](https://user-images.githubusercontent.com/25589559/100987427-cd1ec880-354e-11eb-955b-5cd8078af273.png)

`checkPathsExistence` -> `checkStarted`

so, you have to pass opts to checkPathsExistence, then pass opts from checkPathsExistence to checkStarted

Reply:
following that I'm able to pass opts
![image](https://user-images.githubusercontent.com/25589559/100988046-8e3d4280-354f-11eb-9bf8-ef0f27672b68.png)

Reply:
here you go https://github.com/vvo/selenium-standalone/compare/pass-opts-to-check-started